pretyping/typeclasses.mli

```diff
@@ -93,7 +93,12 @@ val no_goals : evar_filter
 val no_goals_or_obligations : evar_filter
 
 (** Resolvability.
-    Only undefined evars can be marked or checked for resolvability. *)
+    Only undefined evars can be marked or checked for resolvability.
+    They represent TC search roots.
+
+    A resolvable evar is an evar the TC engine may try to solve
+    An unresolvable evar is an evar the TC engine will NOT try to solve
```

Comment:
Comment on the form in passing: the use of an abbreviation `TC` might look a bit cryptic and add more confusion than it helps.